scripts/terraform_schema.py

```diff
@@ -0,0 +1,64 @@
+"""
+Cache Terraform version and schema information in terraform/_schema.json,
+or verify that the cached Terraform information is up-to-date.
+"""
+import argparse
+import json
+import logging
+import os
+import subprocess
+
+from azul.logging import (
+    configure_script_logging,
+)
+
+log = logging.getLogger(__name__)
+project_root = os.environ['project_root']
+terraform_dir = os.path.join(project_root, 'terraform')
+schema_file = os.path.join(terraform_dir, '_schema.json')
+
+
+def terraform_cmd(*args: str) -> str:
```

Comment:
In the other script a similar method is called `terraform_state` so this would imply that this method should be called just `terraform`

Reply:
Standup: Hannes says to rename the functions to `run_terraform` and `run_terraform_state`.